plugin.ts

```diff
@@ -78,4 +89,10 @@ function isLiteral(node: Node): node is Literal {
   return "value" in node && typeof node.value === "string";
 }
 
+interface Paragraph extends Node {}
```

Comment:
This interface doesn't feel strictly necessary, but it is nice to use the type `nodes: (Literal | Paragraph)[]` above instead of broadening it to `nodes: Node[]`.